src/utils.jl

```diff
@@ -185,3 +190,6 @@ function _normal_quantiles_from_ranks!(q, r; α=3//8)
     q .= (r .- α) ./ (n - 2α + 1)
     return q
 end
+
+_val(k) = Val(k)
```

Comment:
Same concerns as above about supporting both `Symbol` and `Val`. I guess it would be reasonable though to constrain it to
```suggestion
_val(k::Symbol) = Val(k)
```